modules/tinymce/src/themes/silver/test/ts/browser/editor/core/UndoRedoTest.ts

```diff
@@ -0,0 +1,58 @@
+import { Keys, UiFinder } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { SugarBody } from '@ephox/sugar';
+import { TinyHooks, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.themes.silver.editor.core.UndoRedoUiTest', () => {
```

Comment:
```suggestion
describe('browser.tinymce.themes.silver.editor.core.UndoRedoTest', () => {
```
Looks like this was missed?